shared/package.json

```diff
@@ -16,6 +16,8 @@
     "tsc-alias": "1.6.11"
   },
   "dependencies": {
+    "@types/change-case": "2.3.1",
```

Comment:
types should be in the "devDependencies"
you need to uninstall the package and then install with -DE flag
Ex – `npm i @types/change-case -DE`